Makefile

```diff
@@ -1119,10 +1119,12 @@ ifdef LANGUAGES
 	mkdir -p $(APPRESOURCESDIR)/lang/mo/
 	cp -pR lang/mo/* $(APPRESOURCESDIR)/lang/mo/
 endif
+ifeq ($(TILES), 0)
 ifeq ($(LOCALIZE), 1)
 	LIBINTL=$$($(CROSS)otool -L $(APPTARGET) | grep libintl | sed -n 's/\(.*\.dylib\).*/\1/p') && if [ -f $$LIBINTL ]; then cp $$LIBINTL $(APPRESOURCESDIR)/; fi; \
```

Comment:
This actually needs to be switched to reference the libintl in the executable directory - see my comment in #49572 - at least to enable command-line execution when desired.